app/javascript/mastodon/features/status/components/detailed_status.js

```diff
@@ -53,7 +53,7 @@ export default class DetailedStatus extends ImmutablePureComponent {
     if (status.get('media_attachments').size > 0) {
       if (status.get('media_attachments').some(item => item.get('type') === 'unknown')) {
         media = <AttachmentList media={status.get('media_attachments')} />;
-      } else if (status.getIn(['media_attachments', 0, 'type']) === 'video') {
+      } else if (['audio', 'video'].indexOf(status.getIn(['media_attachments', 0, 'type'])) !== -1) {
```

Comment:
I'm not actually part of the project, but isn't there `Array.prototype.includes()`? 😁
Here is some documentation: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes

Reply:
`Array.includes()` is only in ES2015+ JS standard, so unless Mastodon is using Babel or some such transposer, it won't work for many browsers. That said, for all I know, it could already be using Babel—I haven't checked the source like I should have...